Null Pointer Exception in StreamMetadataTasks

**Problem description**
```
Caused by: java.lang.NullPointerException: null
        at io.pravega.controller.task.Stream.StreamMetadataTasks.truncate(StreamMetadataTasks.java:333)
        at io.pravega.controller.task.Stream.StreamMetadataTasks.lambda$null$11(StreamMetadataTasks.java:300)
```

This is caused by new log message that was introduced recently:
```
        log.info("Found truncation record for stream {}/{} newRecord time/size: {}/{}", scope, stream,
                                                                                newRecord.getRecordingTime(), newRecord.getRecordingSize());
```
The record to use here should be `truncationRecord` instead of `newRecord`.

**Problem location**
StreamMetadataTasks.java

**Suggestions for an improvement**
Fix the log

--- a/controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java
+++ b/controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java
@@ -329,8 +329,8 @@ public class StreamMetadataTasks extends TaskBase {
             log.info("No suitable truncation record found, per retention policy for stream {}/{}", scope, stream);
             return CompletableFuture.completedFuture(null);
         }
-        log.info("Found truncation record for stream {}/{} newRecord time/size: {}/{}", scope, stream,
-                                                                                newRecord.getRecordingTime(), newRecord.getRecordingSize());
+        log.info("Found truncation record for stream {}/{} truncationRecord time/size: {}/{}", scope, stream,
+                truncationRecord.get().getRecordingTime(), truncationRecord.get().getRecordingSize());
         return streamMetadataStore.getStreamCutRecord(scope, stream, truncationRecord.get(), context, executor)
                    .thenCompose(streamCutRecord -> startTruncation(scope, stream, streamCutRecord.getStreamCut(), context, requestId))
                    .thenCompose(started -> {

